src/core.js

```diff
@@ -425,6 +425,16 @@ jQuery.extend({
 	support: support
 });
 
+// jsHint would error on this code due to the Symbol not being defined in ES5.
+// Defining this global in .jshintrc would create a danger of using the global
+// unguarded in another place, it seems safer to just disable jsHint for these
+// three lines.
+/* jshint ignore: start */
+if ( typeof Symbol === "function" && Symbol.iterator ) {
```

Comment:
> If there were an environment that implemented `Symbol`, but hadn't implemented `Symbol.iterator` yet, you'd end up with `jQuery.fn.undefined = undefined`.

Correct. Or perhaps even something a little weirder, if `Symbol.iterator` is partially implemented (e.g., by incomplete polyfill).

> If you don't care about that, then it's all good and I agree.

Correct. I don't care about that, and here's why (edit mine):

> I don't know of any runtimes [supported by jQuery] that match that